source/install/install-kubernetes-mattermost.rst

```diff
@@ -1,13 +1,20 @@
 .. _install-kubernetes-mattermost:
 
-Deploy a Mattermost Installation
+Deploying a Mattermost Installation
 ============================
 
-Deploying a Mattermost installation is very simple and just requires creating and applying a single manifest.
+Deploying a Mattermost installation requires creating and applying a single manifest.
```

Comment:
This was true before I added the secret. If you're using enterprise you'll need to apply two manifests (realize I wrote this originally so my bad)

Reply:
Gotcha. Updated.